packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -17,7 +17,7 @@ import {ComboboxListbox} from './ComboboxListbox';
 import {ComboboxListboxGroup} from './ComboboxListboxGroup';
 import {ComboboxListboxOption} from './ComboboxListboxOption';
 import {getIndexedItems, getGroupedItems} from './helpers';
-import type {GroupItemsProps, Item as ItemType, ItemProps, ItemsProps, ListBoxProps, ComboboxProps} from './types';
+import type {GroupItemsProps, ItemProps, ItemsProps, ListBoxProps, ComboboxProps} from './types';
```

Comment:
non-used type based on the new per component tsconfigs not having some settings we once relied on like strict